runtime/platform.c

```diff
@@ -225,15 +403,14 @@ void caml_mem_unmap(void* mem, uintnat size)
 #define Max_sleep_ns  1000000000 //  1 s
```

Comment:
I dislike the punning between spin counts (`spins`. etc) and nanoseconds (`Max_sleep_ns`, etc.)

Reply:
I'm not sure what there is to do, it is an unfortunate part of the original spin wait code, and not one I particularly want to touch. Though perhaps it could be commented, I suppose.